src/basic/terminal-util.c

```diff
@@ -1278,6 +1278,11 @@ ColorMode get_color_mode(void) {
                         /* We only check for the presence of the variable; value is ignored. */
                         cached_color_mode = COLOR_OFF;
 
+                else if (STRPTR_IN_SET(getenv("COLORTERM"),
+                        "truecolor",
+                        "24bit"))
```

Comment:
The code is OK, but the alignment is wrong (misleading). `"truecolor"` needs to be aligned to the same column that `getenv`, because those are arguments to the same macro call.

Reply:
ok. Done.